src/lib/analytics/analytics.js

```diff
@@ -304,7 +304,7 @@ const patchLogger = () => {
   logger.error = (...args) => {
     const { Unexpected, Network, Human } = ExceptionCategory
     const [logContext, logMessage, eMsg = '', errorObj, extra = {}] = args
-    let { dialogShown, category = Unexpected } = extra
+    let { dialogShown, category = Unexpected, ...context } = extra
```

Comment:
there's a bugfix here: https://github.com/GoodDollar/GoodDAPP/pull/2236
please merge it once it will be merged to the master

Reply:
the bugfix was merged, so you could apply the fix